documentation/docs/getting_started/docker.md

```diff
@@ -4,20 +4,117 @@ keywords:
 - docker
 ---
 
-# Docker
+# Setting up `Hornet` with `Chronicle` as INX plugin for permanent data storage and Tangle and Ledger live analytics
 
-## Usage
+This is a guide to set up a `Hornet` node together with `Chronicle` as an INX plugin to persist Tangle data without pruning (make sure to maintain sufficient hard disk storage), gather time series analytics data from the Tangle and the UTXO ledger, and display such data in meaningful ways using Grafana. 
 
-The easiest way to start Chronicle is by using our supplied Dockerfile.
+Once you've cloned the repository from [Github](https://github.com/iotaledger/inx-chronicle.git), switch to the `docker` folder:
+```sh
+cd inx-chronicle/docker
+```
+
+## 1. Create the necessary directories for all `Hornet` and `Chronicle` databases
+
+```sh
+sudo ./create_dirs.sh
```

Comment:
`sudo` being unix-only, should we perhaps leave it out?

Reply:
okay :+1: